1.0

carterv opened the pull request:
## Summary
<!--- Please include a summary of the changes here --->
I went through a bunch of tests and fixed bugs as I found them. I also moved the filter stuff out of collect and into util/project.  Finally, I bumped the version to 1.0. 

## Related Issues
<!--- Please relate an issue using either "Related to #issuenum" or "Closes #issuenum" --->
N/A

## Type of Change
<!--- Please select at least one --->
- [x] Bug fix
- [ ] New feature
- [ ] Breaking change (this change would somehow cause existing functionality to not behave as expected)
- [ ] Other

## Testing
<!--- Please describe testing done on this change --->
* Passed all unit tests
* Walked through all of the features and ensured that they worked as expected. Fixed bugs as I found them. 

## Checklist:
<!--- Please check off the following items by replacing [ ] with [x] ---> 
- [x] My code follows PEP8 style guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

carterv committed:
Code cleanup, refactoring, usability

carterv committed:
Major version bump, extra error handling

carterv committed:
Minor bug fixes

ShaneKent reviewed (commented):
What bug fixes in particular did you apply?

carterv commented:
@ShaneKent HTML export was broken because the colors were moved. Creating a new project and then canceling while a project was already broken had weird behavior. Importing a log twice caused the import process to fail, so I added a handler to that to be more user-friendly. I added some extra error messages to the query UI rather than having it silently fail. There's probably another that I'm forgetting.

ShaneKent reviewed (approved):
Great work.

carterv committed:
Modifications for pypi
